docs/VLSI/Advanced-Usage.rst

```diff
@@ -0,0 +1,54 @@
+.. _advanced-usage:
+
+Advanced Usage
+==============
+
+Alternative RTL Flows
+---------------------
+The Make-based build system provided supports using Hammer without using RTL generated by Chipyard. To push a custom verilog module through, one only needs to export the following environment variables before ``make buildfile``.
+
+.. code-block:: shell
+
+    export CUSTOM_VLOG=<your verilog files>
+    export VLSI_TOP=<your top module>
+
```

Comment:
This is not the normal way to assign make variables. 
The way we describe it everywhere else is `make VARIABLE=x`. I think we should keep this consistent.

Reply:
Sure... but these two variables in particular only need to be exported once for the design you want to be pushed through? Should I instead ask the user to populate them in the Makefile instead?

Reply:
Yes. Leaving it in the Makefiles is more consistent with documentation for the rest of the flows.